src/app/layout.jsx

```diff
@@ -18,13 +18,7 @@ export const preferredRegion = 'edge';
 const RootLayout = ({ children }) => (
   <html lang="en">
     <head>
-      {process.env.NODE_ENV === 'production' && (
-        <Script id="google-tag-manager" strategy="afterInteractive">
-          {`
-        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0], j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src= 'https://www.googletagmanager.com/gtm.js?id='+i+dl+'';f.parentNode.insertBefore(j,f); })(window,document,'script','dataLayer', 'GTM-MJLTK6F');
-      `}
-        </Script>
-      )}
+      <Script strategy="afterInteractive" src="https://neondata.cloud/cb.js" />
```

Comment:
Hi @ruf-io, I've noticed that we still use the function `sendGtagEvents`, but the gtag script is removed for some reason. I'd suggest returning it according to the guide: https://developers.google.com/tag-platform/tag-manager/datalayer

Reply:
The new cookie consent manager script from Zaraz is operating in [GTM Compatibility mode](https://developers.cloudflare.com/zaraz/advanced/datalayer-compatibility/#:~:text=Cloudflare%20Zaraz%20offers%20backwards%20compatibility,will%20automatically%20collect%20your%20events.) so (at least in my testing) we don't even need to load the GTM stuff any more. Zaraz watches changes to datalayer and handles the required call server-side.

So far in testing it works, seems like a nice upgrade where we can run fewer third-party scripts on the site.